hack/run-e2e-kind.sh

```diff
@@ -280,35 +280,46 @@ function kube-test-env-up {
 
     # Start Helm Server
-    echo "Installing Helm Server..."
-    kubectl -n kube-system create serviceaccount tiller
-    kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+    # echo "Installing Helm Server..."
+    # kubectl -n kube-system create serviceaccount tiller
+    # kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
+
+    # echo "Initialize Helm Server..."
+    # helm init --service-account tiller
+    # echo "Wait for Helm Server to complete startup..."
+    # sleep 25
+
+    # echo "Getting Helm Server info..."
+    # tiller_pod=$(kubectl get pods --namespace kube-system | grep tiller | awk '{print $1}')
 
-    echo "Initialize Helm Server..."
-    helm init --service-account tiller
-    echo "Wait for Helm Server to complete startup..."
-    sleep 25
+    # kubectl describe pod ${tiller_pod} -n kube-system
 
-    echo "Getting Helm Server info..."
-    tiller_pod=$(kubectl get pods --namespace kube-system | grep tiller | awk '{print $1}')
+    # Installing helm3
 
-    kubectl describe pod ${tiller_pod} -n kube-system
+    curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+    chmod 700 get_helm.sh
+    ./get_helm.sh
+    sleep 10
 
     helm version 
 
-    cd deployment
+    echo "Installing Podgroup CRD"
+
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/scheduler-plugins/277b6bdec18f8a9e9ccd1bfeaf4b66495bfc6f92/config/crd/bases/scheduling.sigs.k8s.io_podgroups.yaml
+
+    cd deployment/mcad-controller
 
     # start mcad controller
     echo "Starting MCAD Controller..."
     echo "helm install mcad-controller namespace kube-system wait set loglevel=2 set resources.requests.cpu=1000m set resources.requests.memory=1024Mi set resources.limits.cpu=4000m set resources.limits.memory=4096Mi set image.repository=$IMAGE_REPOSITORY_MCAD set image.tag=$IMAGE_TAG_MCAD set image.pullPolicy=$MCAD_IMAGE_PULL_POLICY"
-    helm install mcad-controller --namespace kube-system --wait --set loglevel=2 --set resources.requests.cpu=1000m --set resources.requests.memory=1024Mi --set resources.limits.cpu=4000m --set resources.limits.memory=4096Mi --set image.repository=$IMAGE_REPOSITORY_MCAD --set image.tag=$IMAGE_TAG_MCAD --set image.pullPolicy=$MCAD_IMAGE_PULL_POLICY
+    helm upgrade --install mcad-controller .  --namespace kube-system --wait --set loglevel=2 --set resources.requests.cpu=1000m --set resources.requests.memory=1024Mi --set resources.limits.cpu=4000m --set resources.limits.memory=4096Mi --set configMap.name=mcad-controller-configmap --set configMap.podCreationTimeout='"120000"' --set configMap.quotaEnabled='"false"' --set coscheduler.rbac.apiGroup=scheduling.sigs.k8s.io --set coscheduler.rbac.resource=podgroups --set image.repository=$IMAGE_REPOSITORY_MCAD --set image.tag=$IMAGE_TAG_MCAD --set image.pullPolicy=$MCAD_IMAGE_PULL_POLICY
```

Comment:
@metalcycling this is Travis infrastructure, this flag is on by default? if no then how are the tests passing?

Reply:
That is the thing. I don't know how that is supposed to be working given that the default value is `preemptionEnabled: '"false"'` (that is in line 50 of the `deployment/mcad-controller/values.yaml` Helm deployment file). The tests are passing, but that probably means that functionality is not being tested correctly or that value is not used at all. I can see that value is nowhere in the `queuejob_controller_ex.go` where all the preemption stuff is. So I don't know what we should do. If it doesn't change anything, I would still add it there. I can see it is used in `./pkg/controller/quota/quotamanager/quota_rest_manager.go` so it is not affecting our current work, but it may have issues in the quota management work. So we should add it.